src/components/NftCollectionTransfers/index.tsx

```diff
@@ -101,21 +104,46 @@ const TransferTableRow: FC<{
   // eslint-disable-next-line react-hooks/exhaustive-deps
   const timeRelativeBlockCreate = useMemo(() => dayjs(item.transaction.block_timestamp).fromNow(), [now])
 
+  const renderCover = () => {
+    const cell = item.item?.cell
+    const standard = item.item?.standard
+
+    if (standard === 'spore' && cell && cell.data) {
+      const sporeData = parseSporeCellData(cell.data)
+      if (sporeData.contentType.slice(0, 5) === 'image') {
+        const base64data = Base64.fromUint8Array(hexToBytes(`0x${sporeData.content}`))
+
+        return (
+          <img
+            src={`data:${sporeData.contentType};base64,${base64data}`}
+            alt="cover"
+            loading="lazy"
+            className={styles.icon}
+          />
+        )
+      }
+    }
+
+    if (coverUrl) {
+      return (
+        <img
+          src={`${patchMibaoImg(coverUrl)}?size=small&tid=${item.item.token_id}`}
+          alt="cover"
+          loading="lazy"
+          className={styles.icon}
+          onError={handleNftImgError}
+        />
+      )
+    }
+
+    return <img src="/images/nft_placeholder.png" alt="cover" loading="lazy" className={styles.icon} />
```

Comment:
It can be simplified by updating `src` instead of returning different components